include/net/net_context.h

```diff
@@ -946,6 +951,25 @@ int net_context_recv(struct net_context *context,
 		     s32_t timeout,
 		     void *user_data);
 
+struct net_conn;
+/**
+ * @brief Deliver the incoming packet through the recv_cb of the net_context
+ *        to the upper layers
+ *
+ * @param conn		Network connection
+ * @param pkt		Network packet
+ * @param ip_hdr	Pointer to IP header, optional
+ * @param proto_hdr	Pointer to transport layer protocol header, optional
+ * @param user_data	User data passed as an argument
+ *
+ * @return NET_OK	if the packet is consumed through the recv_cb
+ *         NET_DROP	if the recv_cb isn't set
+ */
+enum net_verdict net_context_packet_received(struct net_conn *conn,
```

Comment:
The documentation is ok to add but this function should stay at net_private.h as we do not want to expose it to be called by some application. Eventually we will make net_context.h private but I think we are not yet there.

Reply:
OK, fixed and thanks.